packages/react/package.json

```diff
@@ -107,6 +107,7 @@
     "magic-sdk": "^8.1.1",
     "mime": "^3.0.0",
     "react-cool-dimensions": "^2.0.7",
+    "react-dropzone": "^14.2.2",
```

Comment:
oof do we need this, would like to not force dependencies like this on devs

Reply:
It's a nicer DX - also bundle size is quite small, checked it before I added

Reply:
We can phase it out over time, this is a minimal flow